data/110/869/801/1/1108698011.geojson

```diff
@@ -38,6 +38,9 @@
     "name:eng_x_preferred":[
         "Lapu-lapu City"
     ],
+    "name:fra_x_preferred":[
+        "Lapu-lapu City"
```

Comment:
Really?

Reply:
Hmm.. another questionable translation. I'll follow-up about this one, too.